scss/mixins/_progress.scss

```diff
@@ -9,6 +9,11 @@
     background-color: $color;
   }
 
+  // IE10+, Microsoft Edge
+  &[value] {
```

Comment:
it feels "cleaner" to me not to - as it follows the common advice that's floating around about IE using color rather than background-color, and the basic example from https://msdn.microsoft.com/en-us/library/windows/apps/jj651676.aspx 

have to admit though, as soon as i pushed this new version, i started pondering that myself. using `::-ms-fill` would at least be consistent with the use of pseudos for other engines...